public/css/category.css

```diff
@@ -0,0 +1,67 @@
+@import url('https://framer-ui.netlify.app/components/framer.css');
+
+:root{
+    --navbar-bg:#2C2929;
+    --img-bg:#2F3136;
+    --img-text:#f1f2f3;
+    --img-banner:#4F46E5;
+}
+
+.navbar-bg{
+    background: var(--navbar-bg);
+}
+
+.nav-zindex{
+    z-index: 9999;
+}
+.img-category{
+    min-width: 200px;
+    width: 19.5rem;
+    height: 100%;
+    border-radius: 10px;
+    background: var(--img-bg);
+    cursor: pointer;
+    min-height: 15.625rem;
```

Comment:
I am not sure if you need to use 3 decimal points precision. But if you need it, then it's fine.